src/sentry/static/sentry/app/routes.jsx

```diff
@@ -116,14 +117,21 @@ function appendTrailingSlash(nextState, replaceState) {
   }
 }
 
-const accountSettingsRoutes = [
-  <IndexRedirect key="account-settings-index" to="notifications/" />,
+const accountSettingsDetailedRoutes = [
```

Comment:
nit: I would just nest this inline inside of `notifications/` route. The other route arrays were so we could re-use route trees, which we aren't doing with these.